### DISCUSSION TITLE
Unable to verify school email

### DISCUSSION THREAD

**Discussion by PrpleMoon** [community/community#69692] [GitHub Education]
### Hi everyone,

My school doesn't allow emails from external, which means that the verification email will not appear in my email. Will I still be accepted if I use a personal email?

**Comment by santoshvandari** [comment]
If your school email doesn't allow emails from external sources and you are unable to receive the verification email from GitHub Education, you can consider using a personal email for verification. GitHub Education typically allows users to verify their student status using a personal email address if their school email is not suitable for verification.

Here are the steps you can follow:

- Visit the GitHub Education website and start the verification process.

- When prompted for your email address, use your personal email address.

- Complete the verification process as instructed.

Using your personal email for verification should allow you to access GitHub Education benefits. However, please note that GitHub's policies and procedures may change over time, so it's a good idea to refer to the GitHub Education website or contact GitHub support for the most up-to-date information on their verification process and policies regarding email verification.
